cmd/fluent-bit/loki.go

```diff
@@ -68,6 +68,26 @@ func (l *loki) sendRecord(r map[interface{}]interface{}, ts time.Time) error {
 	return l.client.Handle(lbs, ts, line)
 }
 
+// prevent base64-encoding []byte values (default json.Encoder rule) by
+// converting them to strings
+
```

Comment:
I would remove that empty space and add the function name instead such as `toStringSlice prevent....`

Reply:
This comment is related to the next two functions, so I kept it separate.